service_container/definitions.rst

```diff
@@ -32,11 +32,11 @@ There are some helpful methods for working with the service definitions::
     $definition = $container->findDefinition($serviceId);
 
     // add a new "app.number_generator" definitions
-    $definition = new Definition('AppBundle\NumberGenerator');
+    $definition = new Definition(\AppBundle\NumberGenerator::class);
```

Comment:
It's not redundant. Using `::class` in this situation keeps support for IDE refactoring tools (while string representations don't). So using `::class` is *always* better.

I decided to do it this way to make the code block more readable. It contains a lot of methods to interact with definitions. Having a use statement at top of this code block doesn't make much sense and adding the use statement directly above this seemed even more redundant.